src/components/ScenarioParameters/components/ScenarioParametersInputs/GenericTable.js

```diff
@@ -63,24 +65,38 @@ export const GenericTable = ({
   const workspaceId = useWorkspaceId();
   const datasets = useSelector((state) => state.dataset?.list?.data);
   const scenarioId = useSelector((state) => state.scenario?.current?.data?.id);
+  const addRowEnabled = ConfigUtils.getParameterAttribute(parameterData, 'addRowEnabled') ?? false;
 
   const parameterId = parameterData.id;
   const [parameter, setParameter] = useState(parameterValue || {});
 
   const lockId = `${scenarioId}_${parameterId}`;
 
+  const [isDialogOpen, setDialogOpen] = useState(false);
```

Comment:
We might have several dialogs in the future, can you rename `isDialogOpen` and `setDialogOpen` to be more specific? (e.g. `isConfirmRowsDeletionDialogOpen`)

Reply:
DONE